src/tellor_disputables/data.py

```diff
@@ -49,6 +52,20 @@ def get_contract_info(chain_id: int) -> Tuple[Optional[str], Optional[str]]:
         return None, None
 
 
+def get_query_type(q: OracleQuery) -> str:
+    """Get query type from class name"""
+    return type(q).__name__
+
+
+def get_query_id(q: OracleQuery) -> str:
+    """Get query id from OracleQuery"""
+    return str(q.query_id.hex())
+
+
+def decode_reported_value(q: OracleQuery, args: Dict[str, Any]) -> Any:
+    return q.value_type.decode(args["_value"])
```

Comment:
Assumes args will always have a `_value` key. This always the case?

Reply:
corrected in new changes :)